changes.go

```diff
@@ -365,8 +365,7 @@ type QueryChangeOptions struct {
 	QueryOptions
 
 	// The S or start query parameter can be supplied to skip a number of changes from the list.
-	Skip  int `url:"S,omitempty"`
-	Start int `url:"start,omitempty"`
```

Comment:
This is a breaking change here. If the `start` parameter is not filled, it is dropped anyway (regarding `omitempty`.
On the other side, it is consistent to only go with the `S` parameter instead offering both.
What do you think @opalmer and @shurcooL ?

Reply:
I think there is an unexpecting behaviour if both fields are used. This is why I suggested to remove this uncertainty.

Reply:
So here's my opinion:

- Field should be named `Skip` since that mirrors the name used in other option structure.
- Technically speaking `Skip` should actually be named `Start` since that mirrors what Gerrit's own documentation mentions. However, we shouldn't change that now because that would break a whole lot of code.
- For now we should keep `Start` and `Skip` and:
  - Return an error if both are used. @michaeldorner's right that the current situation is a bit unexpected and we don't really know what the behavior is as a result. However....
  - If `Start` is used and `Skip` is not, set `Skip = Start` (and only have the json tags on `Skip`). That will keep existing code from breaking immediately.
  - Go does have a way of documenting deprecation: https://blog.golang.org/godoc-documenting-go-code
   > Sometimes a struct field, function, type, or even a whole package becomes redundant or unnecessary, but must be kept for compatibility with existing programs. To signal that an identifier should not be used, add a paragraph to its doc comment that begins with "Deprecated:" followed by some information about the deprecation.

The above suggestions seems like it would fix the odd behaviors, not break existing code unless that code is already broken and make it clear what the path for migration is. I'd probably be more inclined to break it if needed to be changed everywhere or there were no other options. 

Thoughts?

Reply:
Sounds good! @michaeldorner Would you mind changing the PR in this way @opalmer suggested?

Reply:
Not yet, but on my desk. :)